src/lib/Components/Inbox/Conversations/SendConversationMessage.ts

```diff
@@ -1,20 +1,29 @@
 import { commitMutation, Environment, graphql, MutationConfig, RecordSourceSelectorProxy } from "react-relay"
 import { ConnectionHandler } from "relay-runtime"
-import RelayProps from "./relayInterfaces"
+
+interface Conversation {
+  last_message_id: string
+  id: string
+  __id: string
+  from: {
+    email: string
+  }
+}
 
 export function sendConversationMessage(
   environment: Environment,
-  conversation: RelayProps["me"]["conversation"],
+  conversation: Conversation,
   text: string,
   onCompleted: MutationConfig["onCompleted"],
   onError: MutationConfig["onError"]
 ) {
-  const lastMessage = conversation.messages.edges[conversation.messages.edges.length - 1].node
-
   const storeUpdater = (store: RecordSourceSelectorProxy) => {
     const mutationPayload = store.getRootField("sendConversationMessage")
     const newMessageEdge = mutationPayload.getLinkedRecord("messageEdge")
-    const connection = ConnectionHandler.getConnection(store.get(conversation.__id), "Conversation_messages")
+    const conversationStore = store.get(conversation.__id)
+
+    // TODO: Why is this not working?
+    const connection = ConnectionHandler.getConnection(conversationStore, "Messages_messages")
```

Comment:
For some reason, after the refactor, I'm unable to figure out how to get a handle to the connection. I tried `Conversation_messages`, etc. Not sure I understand the magic here yet.

Reply:
This should be the `key` that the connection in question is configured with, so this does look right 🤔 Is the return value `null`?

Reply:
Nope, `undefined`